### ISSUE TITLE
Random freezing on Godot 4.0

### ISSUE THREAD

**Issue by zeroxoneafour** [godotengine/godot#50143]
### Godot version

4.0.dev (6e3f47983 and cb4e42155, when not stripped)

### System information

Arch Linux, Vulkan, Official Nvidia Drivers

### Issue description

When I am using the editor, usually in the first minute it completely freezes up. Occasionally I will not be able to move my mouse or really interact with my computer at all, but that resolves itself in a few seconds. However, the editor remains frozen. Sometimes I must use a tty to kill it, and other times I can use a terminal emulator from the desktop environment.

### Steps to reproduce

Open the editor on Linux. I have tested it with the downloadable binaries and it is not an issue, so it's probably 4.0 only.

**Comment by Chaosus** [comment]
Instead of `4.0dev` - please, specify the commit hash of the development build and specify a computer configuration: CPU, GPU, and its driver version. 

Many of Godot's developers work on Linux and if such a problem appeared it may be fixed a long time ago, and it may be your hardware problem.

**Comment by Calinou** [comment]
Make sure to use a recent enough commit since https://github.com/godotengine/godot/issues/41614 was recently fixed.

**Comment by zeroxoneafour** [comment]
It seems to only be a problem on non-stripped binaries on the most recent version. In the older version, single window mode worked.

**Comment by ivalylo** [comment]
I have similar issue on Windows 10 and NVIDIA video card. Happens rarely. I tracked it to vkQueueSubmit(). It may fail with VK_ERROR_DEVICE_LOST, and Godot can't recover from that.

**Comment by clayjohn** [comment]
@zeroxoneafour Are you still running into issues with random freezing in Godot 4?

**Comment by zeroxoneafour** [comment]
It seems to work completely fine now.

**Comment by clayjohn** [comment]
> It seems to work completely fine now.

Then I will close this as completed :)

**Comment by ArteFlamel** [comment]
Can confirm that this is still an issue, using Windows 11 though.

Tried 3 different projects, 2 of which have nothing in them besides a single script for testing. 

Godot 4.2.1-stable_win64 fresh installed as well as the Steam version as of  3/9/24.

In editor mouse locks and no inputs show, there are still changes being made which is dangerous as in my main project I deleted parts of the project thinking Godot froze. 

While the apparent freeze is ongoing scene, project, debug, editor, and help function, also right click context menu appears correctly.

The effect also can happen immediately after attempting to play a scene. After closing that scene any further attempts to open a scene will fail.
